plugins/neat.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: neat
+spec:
+  caveats: |
+    This plugin depends on the following binaries -
+    `jq`, `yq` and `sponge` (from `moreutils`), which should be installed by the user
+  description: |
+    If you try to `kubectl get` resources you have just created,
+    they be unreadably verbose. `kubectl-neat` cleans up that redundant information.
+    Can be used as a `kubectl get` wrapper, or by piping into it.
+    For example `kubectl neat pod mypod -o yaml` or
```

Comment:
It improves readability by putting examples in their own line, like

> Examples:
    $ kubectl neat pod mypod -o yaml
    $ kubectl get pod mypod -o yaml | kubectl neat

Reply:
Thanks, updated